sandbox/OpenIddict.Sandbox.AspNet.Server/Startup.cs

```diff
@@ -272,6 +298,117 @@ await manager.CreateAsync(new OpenIddictApplicationDescriptor
                     }
                 });
             }
+
+            if (await manager.FindByClientIdAsync("maui") is null)
```

Comment:
The MAUI client uses the ASP.NET Core server project, not the legacy ASP.NET 4.x one. Consider revering all these changes.